release-notes/docs/3.10.0.md

```diff
@@ -0,0 +1,149 @@
+# 3.10.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Supporting remote first-time login](#supporting-remote-first-time-login)
+- [Supporting linked docs](#supporting-linked-docs)
+- [Privacy Policies](#privacy-policies)
+- [More granular replication depth configuration for offline users](#more-granular-replication-depth-configuration-for-offline-users)
+- [Improved geolocation data saved for reports](#improved-geolocation-data-saved-for-reports)
+- [Customizable header tabs icons](#customizable-header-tabs-icons)
+- [Improved performance of processing deleted docs](#improved-performance-of-processing-deleted-docs)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+Sentinel no longer advances its transition processing seq in `_local/sentinel-meta-data`. Instead, it advances the transition processing seq in `_local/transitions-seq`. Additionally, a new document `_local/background-seq` advances the deletion processing seq. 
+
+### Breaking changes
+
+#### Updates to meta databases replication
+
+The replication of users meta databases to the conglomerate, `medic-users-meta` database, is no longer configurable. 
+This task now runs every day, at 2am UTC, and replicates `feedback` and `telemetry` documents to `medic-users-meta` database.
+
+#### Android wrapper update required for new features
+
+Some features in 3.10 are only fully functional while using the latest release of medic-android.
```

Comment:
We should specify which version (when it's released) and ideally specify which features require it.